Display/sensors.py

```diff
@@ -0,0 +1,157 @@
+import os
+import random
+from abc import ABC, abstractmethod
+
+from processsampledata import ProcessSampleData
+from rpi_check import is_on_raspberry_pi
+import constants
+
+
+class SensorsDataABC(ABC):
+
+    @abstractmethod
+    def __init__(self,
+                 pressure_sampling_rate,
+                 pressure_oversampling,
+                 temperature_sampling_rate,
+                 temperature_oversampling):
+        super().__init__()
+
+    @abstractmethod
+    def connected_sensors(self):
+        pass
+
+    @abstractmethod
+    def calibration_pressure_sensor_connected(self):
+        pass
+
+    @abstractmethod
+    def poll(self):
+        pass
+
+
+if is_on_raspberry_pi():
```

Comment:
clever use of an if; does this work?

Reply:
It does, yeah.  I copied some code from the RPi standard library (with attribution) to ID the platform.  It can even negatively identify similar boards like Beaglebone Blacks.